distributed/profile.py

```diff
@@ -66,7 +66,7 @@ def info_frame(frame):
     co = frame.f_code
     line = linecache.getline(co.co_filename, frame.f_lineno, frame.f_globals).lstrip()
     return {
-        "filename": co.co_filename,
+        "filename": co.co_filename.replace(sys.exec_prefix, ""),
```

Comment:
Thanks for the screenshots @ncclementi those are super helpful!

Right now it might look to some like `/lib/python3.9/site-packages/tornado/iostream.py` is starting from the root of the filesystem `/` instead of us chopping off the `sys.exec_prefix` part. One way to avoid that confusion is to prepend a "..." to indicate there is more to the path than what we're displaying

```suggestion
        "filename": co.co_filename.replace(sys.exec_prefix, "..."),
```